yellowbrick/classifier/confusion_matrix.py

```diff
@@ -55,6 +55,8 @@ class ConfusionMatrix(ClassificationScoreVisualizer):
         indicated by that function. It may be used to reorder or select a subset of labels.
         If None, values that appear at least once in y_true or y_pred are used in sorted order.
         Default: None
+        
+    label_mapping : dict, a dictionary in the form {class: encoded_class}
```

Comment:
this docstring needs to be updated, I propose the following:

```
label_mapping : dict or LabelEncoder, default: None 
    When specifying the ``classes`` argument, the input to ``fit()`` and ``score()`` must match the 
    expected labels. If the ``X`` and ``y`` datasets have been encoded prior to training and the 
    labels must be preserved for the visualization, use this argument to provide a mapping from the 
    encoded class to the correct label. Because typically a Scikit-Learn ``LabelEncoder`` is used to 
    perform this operation, you may provide it directly to the class to utilize its fitted encoding. 
```

I would also encourage you to review the `classes : ` docstring to make sure it matches this.